File: The-Basic-13/Swap-String-For-Array-Neg-Values/Swap-String-For-Array-Neg-Values.js
// The Basic 13
// Swap-String-For-Array-Neg-Values
// Given an array of numbers replace any negative values with the string "Dojo"

function swap(arr){
  for(let i = 0; i < arr.length; i++){
    if (arr[i] < 0){ arr[i] = "Dojo" }
  }
}

var myArr = [1, -1, 2, -3, 4, -5, 6]
console.log(`The original array is ${myArr}`)
swap(myArr)
console.log(`The changed array is ${myArr}`)

File: The-Basic-13/Swap-String-For-Array-Neg-Values/Swap-String-For-Array-Neg-Values.py
# The Basic 13
# Swap-String-For-Array-Neg-Values
# Given an array of numbers replace any negative values with the string "Dojo"

def swap(arr):
    for i in range(0, len(arr)):
        if arr[i] < 0:
            arr[i] = "Dojo"

myArr = [1, -1, 2, -3, 4, -5, 6]
print("The original array is {}").format(myArr)
swap(myArr)
print("The changed array is {}").format(myArr)
